packages/yoroi-extension/app/api/ada/lib/storage/bridge/updateTransactions.js

```diff
@@ -2261,6 +2261,9 @@ export function networkTxHeaderToDb(
         Extra: {
           Fee: tx.fee,
           Metadata: tx.metadata,
+          // for backward compatiblity, if the field is not present, we take the tx as valid
+          IsValid: !Object.prototype.hasOwnProperty.call(tx, 'valid_transaction') ||
+            tx.valid_transaction,
```

Comment:
@yushih , I ended up adding this single change because it feels more secure to rather treat any nullish/undefined state as `true`, than checking specifically for whether the field is present or not. E.g. any kind of an accidental rework on the backend that might set this value to `undefined` or `null` when flag is not present is in the realm of possibilities.